src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
Yes, it should just keep the original `crumb` value in that case. This is consistent with `dataCallback`'s current behavior (see [here](https://github.com/getsentry/raven-js/blob/master/src/raven.js#L1346)), aside from the explicit `false` return case.